main.cpp

```diff
@@ -52,85 +54,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: dog
+//  action 1:    the dog barks Loudly
+dog.barkLoudly();
+//  action 2:    the dog runs
+dog.run();
+//  action 3:    the dog eat
+dog.eat();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: phone
+//  action 1:    the phone ringgs
+phone.ring(); 
+//  action 2:    the phone plays music
+phone.playMusic();
+//  action 3:    the phone vibrates
+phone.vibrate();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: key
+//  action 1:    the key opens the lock
+key.openLock();
+//  action 2:    the key closes the lock
+key.closeLock();
+//  action 3:    the key reflects light
+key.reflectLight();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: car
+//  action 1:    the car blinks parking lights
+car.blinkParkingLights();
+//  action 2:    the car accelerates
+car.accelerate();
+//  action 3:    the car heats the driver seat
+car.heatDriverSeat();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: speaker
+//  action 1:    the speaker plays music
+speaker.playMusic();
+//  action 2:    the speaker vibrates
+speaker.vibrate();
+//  action 3:    the blinks the led
+speaker.blinkLed();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: server
+//  action 1:    the server turns its led on
+server.turnLedOn();
+//  action 2:    the server hosts packages
+server.hostPackages();
+//  action 3:    the server turns its fan on
+server.turnFanOn();
 //  7)
+//  Noun: finger
+//  action 1:    the finger pushes the button
+finger.pushButton();
+//  action 2:    the finger senses the temperature
+finger.senseTemperature();
+//  action 3:    the finger wraps completely
+finger.wrapCompletely();
+//  8) eye

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.